### DISCUSSION TITLE
How to understand the `extendable` character of G6?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [antvis/G6#4222] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/12727582/215000240-1849a246-274e-4c8e-83fc-294ce0c23fd0.png)
How to understand the `extendable` character of G6?

**Comment by hustcc** [comment, marked as answer]
See the new Readme: [extendable](https://github.com/antvis/G6?tab=readme-ov-file#-%E7%89%B9%E6%80%A7)
